pymeasure/errors.py

```diff
@@ -31,5 +31,9 @@ class RangeError(Error):
     pass
 
 
+class InstrumentError(RuntimeError):
```

Comment:
Could you use built in Exception types instead?
Does that new Exception help to express an error better?

Using a custom Exception makes Exception handling always more complicated, as the user has to import that Error as well.

Reply:
> Does that new Exception help to express an error better?

In that vein, `InstrumentError` seems not very specific?